docs/nifgen/enums.rst

```diff
@@ -702,6 +702,19 @@ TerminalConfiguration
 
 
 
+Trigger
```

Comment:
We should not be generating these. Or at least not publicly and documented.

Reply:
Removed setting enum to Trigger, which disables the generation of this enum